iptables/iptables.go

```diff
@@ -6,6 +6,19 @@ import (
 	"github.com/coreos/go-iptables/iptables"
 )
 
+func iptablesAppendUnique(ipt *iptables.IPTables, table, chain string, pos int, rulespec ...string) error {
```

Comment:
Doh, that is a mistake, I meant to call it `iptablesPrependUnique`. I can rename it to InsertUnique but then it'd make sense to pass the position as an argument as well.

What do you think?